cpp/betweenness_centrality_module/betweenness_centrality_module.cpp

```diff
@@ -17,9 +17,19 @@ constexpr char const *kArgumentThreads = "threads";
 
 void InsertBCRecord(mgp_graph *graph, mgp_result *result, mgp_memory *memory, const double betweenness_centrality,
                     const int node_id) {
+  auto *vertex = mgp::graph_get_vertex_by_id(graph, mgp_vertex_id{.as_int = static_cast<int64_t>(node_id)}, memory);
+  if (!vertex) {
+    if (mgp::graph_is_transactional(graph)) {
+      throw mg_exception::InvalidIDException();
+    }
+    // In IN_MEMORY_ANALYTICAL mode, vertices/edges may be erased by parallel transactions.
+    return;
+  }
+
```

Comment:
These code is duplicates over all modules... It would be good to respect to respect DRY principle here and abstract this code into a method.

Reply:
Will do

Reply:
Done